test/vega_renderer/examples/turicreate/simple_themed_bar_chart.vl.json

```diff
@@ -169,6 +169,7 @@
         },
         "guide-title": {
           "fill": "#fff",
+          "font": "not-a-font",
```

Comment:
I'm not a huge fan of having this in the spec but I wasn't sure of a better way to test behavior when an unavailable font is specified.

Reply:
I think this is fine. I agree there isn't really a better way to test the behavior (at the spec level) when an invalid or missing font is specified.